README.md

```diff
@@ -2,9 +2,9 @@
 
 <div style="float: right; margin-left: 30px;"><img title="The EMP logo was designed by Eamonn Maguire of Antarctic Design." style="float: right;margin-left: 30px;" src="https://upload.wikimedia.org/wikipedia/en/4/4f/EMP-green-small.png" align=right /></div>
 
-The Earth Microbiome Project (EMP) is a systematic attempt to characterize global microbial taxonomic and functional diversity for the benefit of the planet and humankind. Most of the data generated to this point are from 16S rRNA amplicon sequencing - the majority of these data are described, published ([article](http://doi.org/10.1038/nature24621)), and are referred to as EMP 16S Release 1. The project also includes additional 16S data, as well as data from 18S and ITS amplicon sequencing, shotgun metagenomic sequencing, and metabolomic profiling - referred to as the EMP Multi-omics. For more information about the EMP -- people, publications, news, protocols and standards, and more -- please see the [EMP website](http://earthmicrobiome.org/).
+The Earth Microbiome Project (EMP) is a systematic attempt to characterize global microbial taxonomic and functional diversity for the benefit of the planet and humankind. Most of the data generated to this point are from 16S rRNA amplicon sequencing - the majority of these data are described, published ([article](http://doi.org/10.1038/nature24621)), and are referred to as EMP 16S Release 1. The EMP also includes data from 18S and ITS amplicon sequencing, shotgun metagenomic sequencing, and metabolomics profiling. For more information about the EMP -- people, publications, news, protocols and standards, and more -- please see the [EMP website](http://earthmicrobiome.org/).
```

Comment:
I think everything after the first period should be deleted here as (a) it might be confusing to readers as here you only mention release 1 and (b) it seems like the line below is more descriptive and has both releases.